kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandler.java

```diff
@@ -211,6 +211,12 @@
 @Getter
 public class KafkaRequestHandler extends KafkaCommandDecoder {
     private static final int THROTTLE_TIME_MS = 10;
+    /**
+     * Request timeout for writes of the TXMARKERS
+     * Writing the TXMARKERS require recovery of the
+     * transactions on the PartitionLog at it may take much time.
+     */
+    private static final int WRITE_TXN_MARKERS_TIMEOUT = 120000;
```

Comment:
Why do we need to use this dedicated timeout?

Reply:
Thanks, I'll close this PR first.